addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -337,6 +334,20 @@ var ImageWidget = SearchableMediaWidget.extend({
             this.search(this.$('.o_we_search').val() || '');
         }
     },
+    /**
+     * Renders the existing attachments and returns the result as a string.
+     *
+     * @param {Object[]} rows
+     * @param {boolean} withEffect
+     * @returns {string}
+     */
+    _renderExisting: function (rows, withEffect) {
+        return QWeb.render(this.existingAttachmentsTemplate, {
```

Comment:
I would define the property explicitely to `undefined` in the `FileWidget` abstraction.

Reply:
ok